HW_03_Ivanovych/task04.py

```diff
@@ -0,0 +1,11 @@
+#Calculate the area and perimeter of the circle over an entered radius
+import math
+
+
+radius = int(input('Enter radius of the circle'))
+
+area = math.pi * (radius ** 2)
+perimeter = (2 * (math.pi * radius))
```

Comment:
Are the braces necessary in these two lines?

Reply:
taking away brackets